src/interfaces/IFeeCollector.sol

```diff
@@ -8,70 +8,54 @@ interface IFeeCollector {
-     * @param _client A client operator address.
-     * @param _token The token address to check.
-     * @return balance The balance for the specified token for the specified client operator.
-     */
+    /// @notice Returns the balance for the specified token for the specified client operator.
+    /// @dev Example: balances[client][token] = clientBalance
+    /// @param _client A client operator address.
+    /// @param _token The token address to check.
+    /// @return balance The balance for the specified token for the specified client operator.
     function balances(address _client, address _token) external view returns (uint256);
 
-    /**
-     * @notice Returns the take rate of the specified client operator; the percentage of the client rate that the operator keeps.
-     * @param _client A client operator address.
-     * @return clientTakeRate The percentage of the client rate that the operator keeps.
-     */
-    function clientTakeRates(address _client) external view returns (uint256);
-
-    /**
-     * @notice Collects fees from Position contracts when collateral is added.
-     * @param _client The address where a client operator will receive protocols fees.
-     * @param _token The token to collect fees in (the collateral token of the calling Position contract).
-     * @param _amt The total amount of fees to collect.
-     */
+    /// @notice Collects fees from Position contracts when collateral is added.
+    /// @param _client The address where a client operator will receive protocols fees.
+    /// @param _token The token to collect fees in (the collateral token of the calling Position contract).
+    /// @param _amt The total amount of fees to collect.
     function collectFees(address _client, address _token, uint256 _amt, uint256 _clientFee) external payable;
-    /**
-     * @notice Withdraw collected fees from this contract.
-     * @param _token The token address to withdraw.
-     */
+
+    /// @notice Withdraw collected fees from this contract.
+    /// @param _token The token address to withdraw.
     function clientWithdraw(address _token) external payable;
 
-    /**
-     * @notice Allows clients to set the percentage of the clientRate they will receive each revenue-generating tx.
-     *         Amounts less than 100 will give the calling client's users a protocol fee discount:
-     *         clientPercentOfProtocolFee = clientRate * _clientTakeRate
-     *         userPercentOfProtocolFee =  clientRate * (1 - _clientTakeRate)
-     *         clientFee = protocolFee * clientPercentOfProtocolFee
-     *         userSavings = protocolFee * userPercentOfProtocolFee
-     * @param _clientTakeRate The percentage of the clientRate the client will receive each revenue-generating tx (100 = 100%).
-     */
+    /// @notice Allows clients to set the percentage of the clientRate they will receive each revenue-generating tx.
```

Comment:
`///` is meant for single line comments only.

I think it makes sense to remove the calculations from here, since they are detailed in the docs.

```solidity
/// @notice Allows clients to set the percentage of clientRate they receive each revenue-generating tx.
/// @dev Amounts less than 100 will give the calling client's users a protocol fee discount.
/// @param _clientTakeRate The percentage of clientRate the client receives (100 = 100%).
```